### PR TITLE
Add hardcoded JSON comments data

### PR DESCRIPTION
**Pull request by g-lilian**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -13,20 +13,39 @@
...
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Lilian!");
+    String json = convertToJsonUsingGson(messages);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+    /**
```
Comment (123cristen) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:43]: fix the indent here :)

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -13,20 +13,39 @@
...
+   * Converts a Java ArrayList<String> into a JSON string using the Gson library.
+   */
+  private String convertToJsonUsingGson(ArrayList<String> listOfStrings) {
+    Gson gson = new Gson();
```
Comment (123cristen) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:47]: Consider moving this to the class constructor and keeping it as a class instance variable, so you don't have to reconstruct it every time :)

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (g-lilian) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:43]: Fixed!

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (g-lilian) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:47]: Thanks for the tip

**Pull request merged**